mesh/pncmesh.hpp

```diff
@@ -365,7 +365,7 @@ class ParNCMesh : public NCMesh
       void Load(std::istream &is);
       void Decode(Array<int> &elements) const;
 
-      void SetNCMesh(NCMesh *ncmesh) { this->ncmesh = ncmesh; }
+      void SetNCMesh(NCMesh *ncmesh_) { this->ncmesh = ncmesh_; }
```

Comment:
We could also simply name it `mesh`

Reply:
I'm not sure `mesh` is a good name for an `NCMesh` object (`NCMesh` isn't really a "mesh", it's like an extension)